test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAnAppWithTransitiveProjectRefs.cs

```diff
@@ -28,7 +28,7 @@ public void It_builds_the_project_successfully()
             // (TestApp transitively depends on AuxLibrary)
 
             var testAsset = _testAssetsManager
-                .CopyTestAsset("AppWithTransitiveProjectRefs")
+                .CopyTestAsset("AppWithTransitiveProjectRefs", "BuildAppWithTransitiveProjectRef")
```

Comment:
I'm not sure why you needed to add this TestAsset here?  I don't see it used in the test below.